wolfcrypt/src/aes.c

```diff
@@ -4331,7 +4331,7 @@ int wc_AesGcmSetKey(Aes* aes, const byte* key, word32 len)
     }
 #endif
     XMEMSET(iv, 0, AES_BLOCK_SIZE);
-    ret = wc_AesSetKeyLocal(aes, key, len, iv, AES_ENCRYPTION, 0);
+    ret = wc_AesSetKey(aes, key, len, iv, AES_ENCRYPTION);
```

Comment:
Why is this being changed?  Were we explicitly ignoring odd key sizes before with the Local ,,,,0 call?

Reply:
@todd, This was broken a few weeks back in PR #3388. I am reverting to the previous behavior. Sean changes it to use `wc_AesSetKeyLocal ` because the key len check is already done above. The problem is that none of the hardware AES GCM crypto supports the `wc_AesSetKeyLocal`.

Reply:
@toddouska , that PR #3388 reference is here: https://github.com/wolfSSL/wolfssl/pull/3388/files#diff-967c07a1f90d73726a87dc50843985b3ef0f3cbe6d6b05d4ab1f35145d0a765fR4330